README.md

```diff
@@ -26,9 +26,15 @@ Development tasks are managed in the github issues for this repository. When you
 
 ### Pull requests
 
-When you're ready to submit your code, open a pull request with "Closes #X" to link the relavant issue. When your PR is approved by at least one maintainer it is ready to submit.
+When you're ready to submit your code, open a pull request with "Closes #X" to link the relavant issue.
 
-It's easy for the intention of code review comments to be unclear or get misinterpreted. To help with communication, reviewers are encouraged to use [conventional comments](https://conventionalcomments.org/) and explicitly indicate that comments are `(blocking)`, where the discussion must be resolved for PR to be approved, or `(non-blocking)` where resolving the discussion is optional for the implementer.
+It's easy for the intention of code review comments to be unclear or get misinterpreted. To help with communication, reviewers are encouraged to use [conventional comments](https://conventionalcomments.org/) and explicitly indicate that comments are `(blocking)`, where the discussion must be resolved for PR to be merged, or `(non-blocking)` where resolving the discussion is optional for the implementer.
+
+#### Approval and merging
+
+Reviewers should grant approval if they do not feel additional review is necessary before merging. This does not necessarily mean no more changes are required before merging, but that any further changes are expected to be minor enough to not require review.
+
+If the pull request does not require additional changes, the reviewer should merge it immediately after approving. Otherwise, once they have addressed all comments marked `(blocking)` or `nit`, the pull request author should either merge if able or re-request review and merging from a maintainer if not. Authors are encouraged to at least reply to `(non-blocking)` and `(if-minor)` comments if they do not address them with code changes.
```

Comment:
question: will "the reviewer should merge it" be our standard moving forward? I feel the author has been the one merging so far

Reply:
I think it should be that the reviewer merges it unless there are minor things for the author to change post-approval. In open source it's generally more common for maintainers to do all merging.

Reply:
Thanks for clarifying! Sounds good to me